microprofile-fault-tolerance/src/test/java/org/jboss/eap/qe/microprofile/fault/tolerance/util/MicroProfileFaultToleranceServerConfiguration.java

```diff
@@ -0,0 +1,130 @@
+package org.jboss.eap.qe.microprofile.fault.tolerance.util;
+
+import java.io.IOException;
+
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientHelper;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientProvider;
+import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientRelatedException;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+/**
+ * Utility class enabling and disabling MP Fault Tolerance (which is disabled by default)
+ */
+public class MicroProfileFaultToleranceServerConfiguration {
+
+    private static final Address MICROPROFILE_FAULT_TOLERANCE_EXTENSION_ADDRESS = Address
+            .extension("org.wildfly.extension.microprofile.fault-tolerance-smallrye");
+    private static final Address MICROPROFILE_FAULT_TOLERANCE_SUBSYSTEM_ADDRESS = Address
+            .subsystem("microprofile-fault-tolerance-smallrye");
+
+    /**
+     * Enable fault tolerance extension and subsystem.
+     *
+     * @throws ManagementClientRelatedException Wraps exceptions thrown by the internal operation executed by
+     *         {@link OnlineManagementClient} API
+     */
+    public static void enableFaultTolerance() throws ManagementClientRelatedException {
+        try (OnlineManagementClient client = ManagementClientProvider.onlineStandalone()) {
+            enableFaultTolerance(client);
+        } catch (Exception ex) {
+            throw new ManagementClientRelatedException(
+                    "Enabling Fault Tolerance failed. This might crash other tests.", ex);
+        }
+    }
+
+    /**
+     * Enable fault tolerance extension and subsystem.
+     *
+     * @param client {@link OnlineManagementClient} instance used to execute the command
+     * @throws ManagementClientRelatedException Wraps exceptions thrown by the internal operation executed by
+     *         {@link OnlineManagementClient} API
+     */
+    public static void enableFaultTolerance(OnlineManagementClient client) throws ManagementClientRelatedException {
+        if (!faultToleranceExtensionExists(client)) {
+            ManagementClientHelper.executeCliCommand(client, MICROPROFILE_FAULT_TOLERANCE_EXTENSION_ADDRESS + ":add()");
```

Comment:
I didn't try to run this, but using "+" on `Address` and `String` actually works?

Reply:
Yes, it builds CLI command like path to resource:
```
    @Override
    public String toString() {
        if (address.isEmpty()) {
            return "/";
        }

        StringBuilder result = new StringBuilder();
        for (StringPair addressElement : address) {
            result.append("/").append(addressElement.key).append("=").append(addressElement.value);
        }
        return result.toString();
    }
```